rero_ils/modules/collections/api.py

```diff
@@ -89,16 +89,18 @@ class CollectionsIndexer(IlsRecordsIndexer):
 
     record_cls = Collection
 
-    def index(self, record):
-        """Indexing a record."""
+    def after_index_record(self, record):
```

Comment:
This will be removed as we will remove collection information from the document index.

Reply:
ok deleted.